akka-docs/rst/java/stream/stream-graphs.rst

```diff
@@ -235,6 +235,9 @@ Cycles in bounded stream topologies need special considerations to avoid potenti
 This section shows several examples of problems that can arise from the presence of feedback arcs in stream processing
 graphs.
 
+In the following examples runnable graphs are created but do not run because each have some issue and will deadlock after start.
+``Source`` variable is not defined as the nature and number of element does not matter for described problems.
```

Comment:
If you have another PR I think it does not hurt to fix there, falls under "boy-scout" IMHO and does not deserve a dedicated ticket (we have tons of tickets already :) ).